SPEC.md

```diff
@@ -1295,14 +1302,17 @@ Example: string_to_file.wdl
 version 1.1
 
 workflow string_to_file {
-  String path1 = "/path/to/file"
-  File path2 = "/path/to/file"
+  input {
+    File infile
+  }
+
+  String path1 = "~{infile}"
```

Comment:
This makes the test now depend on file to string conversion as well as string to file conversion. But I don't think the test framework lets you set up actual file fixtures to make this work a different way.

Or could we just use `hello.txt`, since it's available to use in the input JSON?